plugin.video.southpark_unofficial/southpark.py

```diff
@@ -0,0 +1,596 @@
+		self.DEFAULT_IMGDIR   = xbmc.translatePath('special://home/addons/{0}/imgs/'.format(addon_id))
+		self.TEMPORARY_FOLDER = xbmc.translatePath('special://temp/southpark')
+		if not os.path.isdir(self.TEMPORARY_FOLDER):
+			os.mkdir(self.TEMPORARY_FOLDER, 0o755)
+
+class KodiParams(object):
+	"""docstring for KodiParams"""
+	def __init__(self, param_string):
+		super(KodiParams, self).__init__()
+		params = _parameters_string_to_dict(param_string)
+		self.PARAM_MODE         = urllib.unquote_plus(params.get('mode'     , ''))
+		self.PARAM_URL          = urllib.unquote_plus(params.get('url'      , ''))
+		self.PARAM_EP_TITLE     = urllib.unquote_plus(params.get('title'    , ''))
+		self.PARAM_EP_THUMBNAIL = urllib.unquote_plus(params.get('thumbnail', ''))
+
+	def debug(self):
+		log_debug("PARAM_MODE:         {0}".format(self.PARAM_MODE        ))
+		log_debug("PARAM_URL:          {0}".format(self.PARAM_URL         ))
+		log_debug("PARAM_EP_TITLE:     {0}".format(self.PARAM_EP_TITLE    ))
+		log_debug("PARAM_EP_THUMBNAIL: {0}".format(self.PARAM_EP_THUMBNAIL))
+		
+
+class SP_I18N(object):
+	"""South Park plugin I18N"""
+	def __init__(self, addon):
+		super(SP_I18N, self).__init__()
+		self.ERROR_NO_INTERNET_CONNECTION         = _translation(addon, 30004)
+		self.MENU_FEATURED_EPISODES               = _translation(addon, 30005)
+		self.MENU_RANDOM_EPISODE                  = _translation(addon, 30006)
+		self.MENU_SEARCH_EPISODE                  = _translation(addon, 30013)
+		self.MENU_SEASON_EPISODE                  = _translation(addon, 30007)
+		self.OPTIONS_AUDIO_LANGUAGE               = _translation(addon, 30000)
+		self.OPTIONS_ENABLE_CC                    = _translation(addon, 30012)
+		self.OPTIONS_GEOLOCATION                  = _translation(addon, 30001)
+		self.OPTIONS_PLAY_DIRECTLY_RANDOM_EPISODE = _translation(addon, 30015)
+		self.WARNING_BAD_INTERNET_CONNECTION      = _translation(addon, 30010)
+		self.WARNING_BANNED_EPISODE               = _translation(addon, 30011)
+		self.WARNING_GEOBLOCKED                   = _translation(addon, 30002)
+		self.WARNING_LOADING                      = _translation(addon, 30009)
+		self.WARNING_LOADING_RANDOM_EPISODE       = _translation(addon, 30003)
+		self.WARNING_PREMIERE                     = _translation(addon, 30014)
+
+class SP_Options(object):
+	"""South Park plugin Options"""
+	def __init__(self, addon):
+		super(SP_Options, self).__init__()
+		self.addon = addon
+		self.GEO_LOCATIONS   = ["US","UK","ES","DE","IT"]
+		self.AUDIO_AVAILABLE = ["en","es","de"]
+		self.VIDEO_QUALITY   = ["high","low"]
+
+	def debug(self):
+		log_debug("OPTIONS Geolocation          {0}".format(self.geolocation(True)))
+		log_debug("OPTIONS Audio                {0}".format(self.audio(True)))
+		log_debug("OPTIONS Show Subtitles       {0}".format(self.show_subtitles()))
+		log_debug("OPTIONS Play Random Directly {0}".format(self.playrandom()))
```

Comment:
reduce the log spamming

Reply:
i've commented the line 571 & 572, so they will never be called.